File: ejercicios/python/ejercicio034.py
#Python no soporta la estructura switch

opc = input("Ingrese un caracter numerico: ")
if opc == '0':
    num = 0
elif opc == '1':
    num = 1
elif opc == '2':
    num = 2
elif opc == '3':
    num = 3
elif opc == '4':
    num = 4
elif opc == '5':
    num = 5
elif opc == '6':
    num = 6
elif opc == '7':
    num = 7  
elif opc == '8':
    num = 8
elif opc == '9':
    num = 9
else:
	num = -1
	
if num != -1:
	print(num)
else:
	print("caracter no valido")    

File: ejercicios/c++/ejercicio034.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main(){
	char c;
	int numero;
	
	cout << "Ingrese un caracter numerico: ";
	cin >> c;
	
	switch(c){
		case '0':
			numero = 0;
			break;
		case '1':
			numero = 1;
			break;
		case '2':
			numero = 2;
			break;
		case '3':
			numero = 3;
			break;
		case '4':
			numero = 4;
			break;
		case '5':
			numero = 5;
			break;
		case '6':
			numero = 6;
			break;
		case '7':
			numero = 7;
			break;
		case '8':
			numero = 8;
			break;
		case '9':
			numero = 9;
			break;
		default:
			numero = -1;
			break;
	}
	
	if(numero != -1){
		cout << numero << endl;
	}else{
		cout << "caracter no valido" << endl;
	}
	
	return 0;
}
